Frontend/ui-common/src/Engineering/Helpers/Suggestions.purs

```diff
@@ -248,6 +475,47 @@ type SuggestionDefinitions = Array
+  {key : "0e9fbeec2f63098db38830475f1f58e5", value :  { en_us : "won't cancel, please come", ta_in : "ரத்து செய்ய மாட்டேன், தயவுசெய்து வாருங்கள்", kn_in : "ರದ್ದು ಮಾಡುವುದಿಲ್ಲ, ದಯವಿಟ್ಟು ಬನ್ನಿ", hi_in : "मैं रद्द नहीं करूँगा, कृपया आएँ", ml_in : "ക്യാൻസൽ ചെയ്യില്ല, ദയവായി വരുക", bn_in : "আমি ক্যানসেল করছি না, আপনি দয়া করে আসুন" }},
+  {key : "f06fcc3f8c955e54834ab089cebe02cc", value :  { en_us : "Are you on the way?", ta_in : "நீங்கள் வந்துகொண்டுஇருக்கீர்களா?", kn_in : "ನೀವು ದಾರಿಯಲ್ಲಿದ್ದೀರಾ?", hi_in : "क्या आप रास्ते में हैं?", ml_in : "താങ്കൾ വന്നുകൊണ്ടിരിക്കുകയാണോ", bn_in : "আপনি কি আসছেন?" }},
+  {key : "2930acb2ae7ac25629025541fb0d2c72", value :  { en_us : "In traffic, please chat", ta_in : "டிராபிக்-இல் உள்ளேன், தயவுற்று மெசேஜ் செய்யவும்", kn_in : "ಟ್ರಾಫಿಕ್‌, ದಯವಿಟ್ಟು ಚಾಟ್ ಮಾಡಿ", hi_in : "मैं ट्रैफिक में हूं, कृपया चैट करें", ml_in : "ട്രാഫിക്കിലാണ്, ദയവായി ചാറ്റ് ചെയ്യുക", bn_in : "ট্রাফিক আটকে আছি, দয়া করে চ্যাট করুন" }},
+  {key : "e68fecabed0bc6a41d16a28b7239d355", value :  { en_us : "Can't chat, please call", ta_in : "மெசேஜ் செய்யமுடியவில்லை தொலைபேசியில் அழைக்கவும்", kn_in : "ಚಾಟ್ ಮಾಡಲು ಸಾಧ್ಯವಿಲ್ಲ, ದಯವಿಟ್ಟು ಕರೆ ಮಾಡಿ", hi_in : "चैट नहीं कर सकते, कृपया कॉल करें", ml_in : "ചാറ്റ് ചെയ്യാനാവില്ല, ദയവായി കാൾചെയ്യൂ", bn_in : "চ্যাট করা যাবে না, কল করুন" }},
+  {key : "95e946dc1a56fce830c31471435ad95c", value :  { en_us : "Looking for you", ta_in : "உங்களை தேடுகிறேன்", kn_in : "ನಿಮ್ಮನ್ನೆ  ಹುಡುಕುತ್ತಿರುವೆ", hi_in : "मैं आपको ढूंड रहा हूँ, आप कहा हो ?", ml_in : "താങ്കളെ തിരയുകയാണ്", bn_in : "আপনাকে খুঁজছি, আপনি কোথায় ?" }},
+  {key : "e485f3d2667273e342a65077e7538338", value :  { en_us : "I've arrived", ta_in : "நான் வந்துவிட்டேன்", kn_in : "ನಾನು ತಲುಪಿದೆ", hi_in : "मैं लोकेशन पे आ गया हूँ ", ml_in : "ഞാൻ എത്തി", bn_in : "আমি পৌঁছে গেছি" }},
+  {key : "4ecdd92ebb79b82cc3b50f4e2e427edf", value :  { en_us : "Waiting, please come", ta_in : "காத்திருக்கிறேன், தயவுசெய்து வாருங்கள்", kn_in : "ಕಾಯುತ್ತಿರುವೆ, ದಯವಿಟ್ಟು ಬನ್ನಿ", hi_in : "इंतजार कर रहे हैं, कृपया आएं", ml_in : "കാത്തിരിക്കുകയാണ്, ദയവായി വരുക", bn_in : "আমি অপেক্ষা করছি, দয়া করে আসুন" }},
+  {key : "0b295e1ff354fe09337a4fa1670dd0f5", value :  { en_us : "Starting, don't cancel", ta_in : "நான் தொடங்குகிறேன், ரத்து செய்ய வேண்டாம்", kn_in : "ಪ್ರಾರಂಭಿಸಲಾಗುತ್ತಿದೆ, ರದ್ದುಗೊಳಿಸಬೇಡಿ", hi_in : "मैं शुरू कर रहा हूं, रद्द न करें", ml_in : "തുടങ്ങുകയാണ്, ക്യാന്സലചെയ്യരുത്", bn_in : "আমি শুরু করছি, ক্যানসেল করবেন না" }}
+]
+
+getSuggestion :: String -> {en_us :: String, ta_in :: String, kn_in :: String, hi_in :: String, ml_in :: String, bn_in :: String}
+getSuggestion key = do
+  let message = filter (\item -> item.key == key) suggestionsList
+  case (head message) of
+    Just record -> record.value
+    Nothing ->  {en_us : "", ta_in : "", kn_in : "", hi_in : "", ml_in : "", bn_in : "" }
```

Comment:
in case we return this empty values, what will be the affect on the UI?

Reply:
This won't be the case cause the data is static